lib/document/stateTransition/documentTransition/DocumentReplaceTransition.js

```diff
@@ -17,6 +17,7 @@ class DocumentReplaceTransition extends AbstractDocumentTransition {
     delete data.$type;
     delete data.$action;
     delete data.$revision;
+    delete data.$protocolVersion;
```

Comment:
We don't need the protocol version in document transitions since we have it in top-level Documents Batch Transition